instrumentation/akka/akka-actor-2.3/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/akkaactor/AkkaScheduleInstrumentation.java

```diff
@@ -41,20 +43,20 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class ScheduleAdvice {
 
+    @AssignReturned.ToArguments({@ToArgument(2)})
```

Comment:
I think you don't need to add `{}` around `@ToArgument(2)`

Reply:
Fixed in the auto-migration tool so that this doesn't happen anymore in the future:
https://github.com/JonasKunz/indy-advice-transformer/commit/e1f0b495b9516e83e4a457fed0812d9ae36f7266